src/GUI/Overlay/OverlayEventHandler.cpp

```diff
@@ -66,12 +66,13 @@ bool OverlayEventHandler::keyEvent(PlatformKey key, PlatformModifiers mods, bool
         else
             nk_input_key(_context, NK_KEY_RIGHT, keyPressed);
     }
-    return nk_item_is_any_active(_context);
+    return _context->last_widget_state & NK_WIDGET_STATE_MODIFIED ||
+        _context->text_edit.active;
 }
 
 bool OverlayEventHandler::mouseMoveEvent(const PlatformMouseEvent *event) {
     nk_input_motion(_context, event->pos.x, event->pos.y);
-    return nk_item_is_any_active(_context);
+    return _context->last_widget_state & NK_WIDGET_STATE_MODIFIED;
```

Comment:
Can't use the `nk_item_is_any_active` here or the input will be catched just by having the mouse over any nuklear window. Very annoying if your walking around in game and suddenly move the mouse on any debug window ( the party will keep moving cause the release input will be catched by nuklear ).